nomenclature/definitions/variable/technology/electrity-operation.yaml

```diff
@@ -0,0 +1,460 @@
+# Variables needed for plan4EU results in terms of electricity system operation
+# Schedules per power generation technologies, storages and demand-response
+# includes for each technology:
+#   - Active Power
+#   - Operation Cost
+#   - Reserves committed and Inertia provided
+#   - Levels of storages
+# Marginal costs for:
+#   - Demand in ACtive Power
+#   - Demand in reserves
+#   - Demand in inertia
+#   - Flows in transmission lines
+# Demand not served
+
+# Active Power per technologies
+# i.e.  aggregated committed (in general hourly) schedules of plants
+
+Active Power|Electricity|Biomass:
```

Comment:
is "Active power" different from "Generation"? if no, I'm wondering if using the more intuitive term isn't better...

Reply:
We hesitated a lot.... The idea is that if Comillas wants further to include reactive power (as was mentioned in a previous discussion), would be more clear with Active Power. For plan4eu, we don't care
@erikfilias what do you think?